src/lib/mongoBackend/TriggeredSubscription.h

```diff
@@ -78,9 +80,10 @@ class TriggeredSubscription
 
   ~TriggeredSubscription();
 
-  void fillExpression(const std::string& q, const std::string& georel, const std::string& geometry, const std::string& coords);
-
-  std::string toString(const std::string& delimiter);
+  // FIXME P5: This method will cease to exist once geo-stuff is implemented the same way StringFilter was implemented (for Issue #1705)
```

Comment:
'q' has been removed, the other params will be removed also and the function will no lopnger make sense and will be removed.

Now, if we use georelFilterSet() etc or some other mechanism, that I cannot answer to right now. When this part of issue 1705 is implemented (or planned in detail) I will be able to answer that question.

NTC